Tools/rdmp/CommandLine/Gui/ConsoleGuiActivator.cs

```diff
@@ -97,8 +97,11 @@ public override bool YesNo(string text, string caption, out bool chosen)
 
         private void GetDialogDimensions(out int w, out int h)
         {
-            w = Math.Min(80, Application.Top.Frame.Width - 4);
-            h = Math.Min(20, Application.Top.Frame.Height - 2);
+            w = Application.Top?.Frame.Width??0;
```

Comment:
Looks like this checks if Top is null, but not Frame?

Reply:
I think Frame is a `struct`